README.md

```diff
@@ -40,8 +49,8 @@ This repo should contain issues that are tied to the runtime, the class librarie
 
 For other issues, please use the following repos:
 
-- For overall .NET SDK issues, file in the [dotnet/sdk](https://github.com/dotnet/sdk) repo
-- For ASP.NET issues, file in the [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore) repo.
+* For overall .NET SDK issues, file in the [dotnet/sdk](https://github.com/dotnet/sdk) repo
```

Comment:
```suggestion
* For overall .NET SDK issues, file in the [dotnet/sdk](https://github.com/dotnet/sdk) repo.
```

Reply:
you could just include all the repos we list in our new issue picker:
https://github.com/dotnet/runtime/blob/1d025196ae0d0f1e0b17e2e6033234e1937d997b/.github/ISSUE_TEMPLATE/config.yml#L3-L20

Reply:
these links go directly to _their_ issue pickers